src/hooks/useScanResults.ts

```diff
@@ -11,12 +11,14 @@ export const SCAN_RESULT = 'CLAIR_SCAN_RESULT';
 export const SCAN_RESULTS = 'CLAIR_SCAN_RESULTS';
 export const CVE_SCAN_RESULT = 'CVE_SCAN_RESULT';
 export const TEKTON_SCAN_RESULTS = 'TEKTON_SCAN_RESULTS';
+export const SCAN_OUTPUT = 'SCAN_OUTPUT';
 
 export const CVE_SCAN_RESULT_FIELDS = [
   SCAN_RESULT,
   SCAN_RESULTS,
   CVE_SCAN_RESULT,
   TEKTON_SCAN_RESULTS,
+  SCAN_OUTPUT,
```

Comment:
you have not added `TEST_OUTPUT`. Do we not need it?

Reply:
I added and then removed it. Actually, we dont have a block for Test output. And this hook is related to scan results. 
Test output is carrying different details like success, warning and not the vulnerability data.

Reply:
do we not use the `TEST_OUTPUT` data anywhere?

Reply:
no not yet